src/pages/profile.tsx

```diff
@@ -129,30 +115,72 @@ const Profile: NextPage = () => {
     },
     resolver: zodResolver(onboardSchema),
   });
+  const { data: user } = trpc.useQuery(["user.me"], { refetchOnMount: true });
```

Comment:
what does refetchOnMount do?

Reply:
This was Luke's solution to the page not re-rendering when a user navigated to it using the back arrows after saving information iirc